src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/DependencyObjectAvailabilityGenerator.cs

```diff
@@ -90,17 +88,15 @@ internal void Generate(GeneratorExecutionContext context)
 
 						modules = modules.Concat(context.Compilation.SourceModule);
 
-						var bindableTypes = from module in modules
-											from type in module.GlobalNamespace.GetNamespaceTypes()
-											where (
-												type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
-												|| additionalLinkerHintSymbols.Contains(type)
-											)
-											select type;
+						var propertyNames = (from module in modules
+											 from type in module.GlobalNamespace.GetNamespaceTypes()
+											 where (
+												 type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
+												 || additionalLinkerHintSymbols.Contains(type)
+											 )
+											 select LinkerHintsHelpers.GetPropertyAvailableName(type.GetFullMetadataName())).ToArray();
 
-						bindableTypes = bindableTypes.ToArray();
```

Comment:
The `ToArray` call here looked suspicious because `bindableTypes` is still IEnumerable. I moved it to the previous type so that the variable is typed as an array.